accepted/2024/package-update-command.md

```diff
@@ -0,0 +1,238 @@
+# dotnet package update command
+
+- [Olia Gavrysh](https://github.com/OliaG/)
+- Start Date (2024-11-11)
+- [#13372](https://github.com/NuGet/Home/issues/13372)
+- [#4103](https://github.com/NuGet/Home/issues/4103)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+The `dotnet package update` feature is designed to enhance the user experience by enabling .NET developers to address security vulnerabilities, deprecated packages, and outdated versions of their NuGet dependencies with a single command.
+This improvement will, in turn, bolster the overall security and reliability of .NET applications. The feature will build upon the existing NuGet Audit functionality.
+
+## Motivation
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+Even though .NET developers now have the opportunity to learn about vulnerabilities and deprecated dependencies in their NuGet packages and addressing those issues, there are gaps in user experience, especially in the CLI workflows.
+
+### Existing capabilities
+
+Visual Studio
+
+- Has a means to show outdated packages
+- Has a means to show deprecate/vulnerable packages
+- Has a means to update all packages to latest
+- Has a means to update a single package to latest (which can be challenging)
+- Has a vulnerability filter, which in turn can allow you to update all vulnerable packages to latest
+dotnet
+
+CLI
+
+- Has a means to show outdated packages
+- Has a means to show deprecated/vulnerable packages
+- Has a means to update a single package
+- Does not have means to update multiple packages at once
+
+### Gaps in CLI experience
+
+- No means to update to all latest packages on the CLI
+- No means for update heuristics, such as, update my vulnerable packages
+
+This proposal is focused on bringing the CLI experience up to funcional parity with the experience in Visual Studio and make developers more productive by providing a single command to update all packages to the latest version and a way to update all vulnerable packages.
+
+## User experience
```

Comment:
I think it would be nice to have a summary of all features, similar to what you'd expect to see when you run `dotnet package update -h`. The sub-feature examples below are good, but I had an idea for something, and a ctrl-f can't find it in the document. Without re-reading the whole thing top to bottom, I don't have an easy way to figure out if I'm searching for the wrong keyword or if it's something I need to suggest.

And that is that I think a `--dry-run` option would be good. But now I feel like this was originally proposed. Did we remove it for some reason that I can't remember?